dashboard/test/controllers/api/v1/pd/course_facilitators_controller_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class Api::V1::Pd::CourseFacilitatorsControllerTest < ::ActionController::TestCase
+  test 'admins can see course facilitators' do
+    sign_in create(:admin)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :success
+  end
+
+  test 'workshop organizers can see course facilitators' do
+    sign_in create(:workshop_organizer)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :success
+  end
+
+  test 'facilitators cannot see course facilitators' do
+    sign_in create(:facilitator)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :forbidden
+  end
+
+  test 'teachers cannot see course facilitators' do
+    sign_in create(:teacher)
+    get :index, course: Pd::Workshop::COURSES.first
+    assert_response :forbidden
```

Comment:
You can combine these two tests (and other cases with

[create(:teacher), create(:facilitator), create(:student)].each do |user|
sign in, get forbidden
end